lib/configs/end_point_urls.dart

```diff
@@ -0,0 +1,7 @@
+import 'configs.dart';
+
+const discoveryUrl =
```

Comment:
Keep a consistent way to manage constants. Either keep it like this, or use a class do not use both.

Reply:
fixed and used class